json_journal/main.py

```diff
@@ -0,0 +1,26 @@
+from json_journal import data_analysis
```

Comment:
Do we want to package this file? Or is it for local testing? If it is for local/ad-hoc testing then perhaps we

a) don't commit it
b) commit it in a different folder where its purpose is clear

Reply:
done